algorithms/maths/num_perfect_squares.py

```diff
@@ -0,0 +1,36 @@
+"""
+Given an integer num_perfect_squares will return the amount of perfect squares that are required to sum to the specified number.
```

Comment:
is it `minimum amount of perfect squares`?

Reply:
'minimum numbers' would be appropriate.